api-tests/src/test/java/org/openmrs/module/reporting/cohort/definition/evaluator/DrugOrderCohortDefinitionEvaluatorTest.java

```diff
@@ -0,0 +1,185 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.reporting.cohort.definition.evaluator;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import java.util.Date;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openmrs.Cohort;
+import org.openmrs.Concept;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.reporting.cohort.definition.DrugOrderCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.service.CohortDefinitionService;
+import org.openmrs.module.reporting.common.DateUtil;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class DrugOrderCohortDefinitionEvaluatorTest extends BaseModuleContextSensitiveTest {
+  protected static final String TEST_DATA = "org/openmrs/module/reporting/include/ReportTestDataset-openmrs-1.10.xml";
+  private DrugOrderCohortDefinition cohortDefinition;
+
+  @Before
+  public void setup() throws Exception {
+    cohortDefinition = new DrugOrderCohortDefinition();
+    executeDataSet(TEST_DATA);
+  }
+
+  @After
+  public void tearDown() {
+    cohortDefinition = null;
+  }
+  
+  @Test
+  public void evaluate_shouldReturnAllPatientsCurrentlyActiveOnDrugs() throws Exception {
+    cohortDefinition.setOnlyCurrentlyActive(true);
+    Cohort cohort = Context.getService(CohortDefinitionService.class).evaluate(cohortDefinition, null);
+    System.out.println("Currently on Drugs " + cohort.getMemberIds() + " " + cohortDefinition + "===");
+  }
+  
+  @Test
+  public void evaluate_shouldReturnAllPatientsCurrentlyNotActiveOnDrugs() throws Exception {
+    cohortDefinition.setOnlyCurrentlyActive(false);
+    Cohort cohort = Context.getService(CohortDefinitionService.class).evaluate(cohortDefinition, null);
+    System.out.println("Currently Not on Drugs " + cohort.getMemberIds() + " " + cohortDefinition + "===");
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotprintout